test/e2e/util_test.go

```diff
@@ -369,24 +369,28 @@ func cleanupOLM(t *testing.T, namespace string) {
 	var err error
 	err = waitForEmptyList(func() (int, error) {
 		res, err := crc.OperatorsV1alpha1().ClusterServiceVersions(namespace).List(metav1.ListOptions{FieldSelector: nonPersistentCSVFieldSelector})
+		t.Logf("%d %s remaining", len(res.Items), "csvs")
```

Comment:
saw some cases where namespace cleanup failed, this helps to find those instances.